### PR TITLE
add release, and HW badges

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
PR to add release version and Help Wanted Badges to the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [![Build Status](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-mscalendar/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-mscalendar)
 [![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-mscalendar/master.svg)](https://codecov.io/gh/mattermost/mattermost-plugin-mscalendar)
+[![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-mscalendar)](https://github.com/mattermost/mattermost-plugin-mscalendar/releases/latest)
+[![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-mscalendar/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-mscalendar/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
```
Comment (mickmister) [inline README.md:6]: Up for Grabs! Great!

**Comment by mickmister** [comment]
Hmm should we have applied the same pre-release badge discussed [here](https://github.com/mattermost/mattermost-plugin-agenda/pull/41)? Probably not a big deal.

**Comment by jfrerich** [comment]
@larkox, @mickmister - I discovered the ability to have the badge recognize pre-releases also.  Do you guys know when/if there will be a pre-release before an actual release version is created?

**Comment by mickmister** [comment]
@jfrerich I don't think we want to advertise it here so I think we're good with just releases

**Pull request merged**